cmd/ore/azure/create-image-arm.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2018 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package azure
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	cmdCreateImageARM = &cobra.Command{
+		Use:   "create-image-arm",
+		Short: "Create Azure image",
+		Long:  "Create Azure image from a blob url",
+		RunE:  runCreateImageARM,
+	}
+
+	imageName     string
+	blobUrl       string
+	resourceGroup string
+)
+
+func init() {
+	sv := cmdCreateImageARM.Flags().StringVar
+
+	sv(&imageName, "image-name", "", "image name")
+	sv(&blobUrl, "image-blob", "", "source blob url")
+	sv(&resourceGroup, "resource-group", "kola", "resource group name")
+
+	Azure.AddCommand(cmdCreateImageARM)
+}
+
+func runCreateImageARM(cmd *cobra.Command, args []string) error {
```

Comment:
I was really confused for a sec and thought Azure supported ARM.

Reply:
Yeah ARM (Azure Resource Manager) can be a confusing name when looking from our usual scope.